dev/com.ibm.ws.jaxrs.2.0_fat/test-applications/paramconverter/src/com/ibm/ws/jaxrs/fat/paramconverter/ClientTestServlet.java

```diff
@@ -257,7 +257,7 @@ public void testQueryParamSpecialCharacters() throws Exception {
         assertEquals("expected: " + expected + " actual: " + actual, expected, actual);
     }
 
-// Waiting for feature 15234    @Test
+@Test
```

Comment:
the spacing seems off here - maybe it should be:
```suggestion
    @Test
```